### DISCUSSION TITLE
IMAP security options

### DISCUSSION THREAD

**Discussion by bododogge** [creecros/Mailmagik#24] [Ideas]
The addition of IMAP security options such as connection security (SSL/TLS , STARTTLS) and Authentication methods would make this plugin really handy even for secure connections over the Internet.

**Comment by creecros** [comment]
when you say authentication methods, do you mean, like: POP3/IMAP/NNTP ?

**Comment by creecros** [comment]
or would it make sense to have an advanced option that let's a user just input all thier flags manually.

https://www.php.net/manual/en/function.imap-open.php

**Comment by creecros** [comment]
https://github.com/creecros/Mailmagik/commit/6610ecfcaa0cefcba5cb10703599466fabcfab04
